composer.json

```diff
@@ -21,11 +21,10 @@
     "ext-mbstring": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7",
+    "phpunit/phpunit": "^5.7 || ^9.0",
     "php-coveralls/php-coveralls": "^2.1",
     "squizlabs/php_codesniffer": "^3.3",
-    "symfony/process": "~3.4",
```

Comment:
Confirming this was just a stale require?

Reply:
Good call. We actually use it in `StripeMock.php` which apparently attempts to boot up stripe-mock if you aren't already running it?

In our CI we always boot up stripe-mock in a separate step, and I always boot stripe-mock up manually, so I went ahead and removed this file.

Reply:
LGTM - good find!